_posts/2018-03-26-Update-on-the-Meltdown-Spectre-vulnerabilities.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "对“熔断” (Meltdown) 和“幽灵” (Spectre) 漏洞的一些跟进"
+date:   2018-03-26 14:01:00 +0800
+---
+
+今年1月份，一系列名为[“幽灵” (Spectre) 和 “熔断” (Meltdown)](https://fzug.github.io/2018/01/18/protect-fedora-system-meltdown.html) 的漏洞被公之于众。问题的性质决定了解决方案非常复杂，并且需要修复相关代码。x86 架构上对“熔断” (Meltdown) 的最初修复 KPTI 几乎在公开时立即发布了。对“幽灵” (Spectre) 漏洞的修补则更加复杂。当然其他架构也需要审视受影响的状态，并且在需要的情况下做出修复。到现在为止已经过去了相当的时间，那么 Fedora 的修复状况如何呢？
```

Comment:
站内可以使用`{% post_url`这样的指令，而不用写绝对url

Reply:
这个在我的github pages 环境下编译不了。我仍保留绝对地址了。

Reply:
@RobberPhex 是指的 `{{ post.url }}` 这样的写法吗？这个代表的好像是当前页面的链接。

我测试了一下，
```markdown
[“幽灵” (Spectre) 和 “熔断” (Meltdown)](https://fzug.github.io/2018/01/18/protect-fedora-system-meltdown.html)
```
可以写成：
```markdown
[“幽灵” (Spectre) 和 “熔断” (Meltdown)]({{ site.url }}/2018/01/18/protect-fedora-system-meltdown.html)
```
或者：
```markdown
[“幽灵” (Spectre) 和 “熔断” (Meltdown)](/2018/01/18/protect-fedora-system-meltdown.html)
```
都能自动补全完整的链接。

Reply:
这样的：

```
[“幽灵” (Spectre) 和 “熔断” (Meltdown)]({% post_url 2018-01-18-protect-fedora-system-meltdown %})
```

---
Jekyll的官方文档表述如下：

> One major benefit of using the link or post_url tag is link validation. If the link doesn’t exist, Jekyll won’t build your site. This is a good thing, as it will alert you to a broken link so you can fix it (rather than allowing you to build and deploy a site with broken links).